quantum/quantum.h

```diff
@@ -152,7 +152,10 @@ extern layer_state_t default_layer_state;
     #define PIN_ADDRESS(p, offset) _SFR_IO8(ADDRESS_BASE + (p >> PORT_SHIFTER) + offset)
 
     #define pin_t uint8_t
-    #define setPinInput(pin) PIN_ADDRESS(pin, 1) &= ~ _BV(pin & 0xF)
+    #define setPinInput(pin) ({\
+            PIN_ADDRESS(pin, 1) &= ~ _BV(pin & 0xF);\
+            PIN_ADDRESS(pin, 2) &= ~ _BV(pin & 0xF);\
+        })
```

Comment:
The GPIO macros were refactored in #5021 to fix an issue with macro parameter expansion. Change this to the following to resolve the merge conflict:
```c
    #define setPinInput(pin)        (PIN_ADDRESS(pin, 1) &= ~_BV((pin) & 0xF), \
                                     PIN_ADDRESS(pin, 2) &= ~_BV((pin) & 0xF))
```

Reply:
Done, thanks :)